posts/2024-03-14/index.qmd

```diff
@@ -0,0 +1,90 @@
+---
+title: "Data interoperability for single-cell analysis"
+author: "Lexanomics"
+date: "2024-03-14"
+execute:
+    warning: false
+    freeze: auto
+categories: [debbuging new]
```

Comment:
Juicy
```suggestion
categories: [code, debbuging new]
```

Reply:
I really dont like the idea of several tags, I think fewer are better